libs/designsystem/item/src/item.component.spec.ts

```diff
@@ -92,6 +92,52 @@ describe('ItemComponent', () => {
         '--inner-padding-end': size('xxs'),
       });
     });
+
+    describe('when rotateIcon is set to true', () => {
+      beforeEach(() => {
+        spectator.setInput('rotateIcon', true);
+      });
+
+      it('should rotate the icon 180deg if the disclosure icon is "arrow-down', () => {
+        spectator.setInput('disclosure', 'arrow-down');
+
+        const icon = spectator.query('kirby-icon');
+
+        expect(icon).toHaveComputedStyle({
+          transform: 'matrix(-1, 0, 0, -1, 0, 0)',
```

Comment:
In the `computed` tab in the browser `rotate(180deg)` is converted to `matrix(-1, 0, 0, -1, 0, 0)`. Is there a better (and less confusing) way to test this?